barretenberg/cpp/src/barretenberg/ecc/curves/secp256r1/secp256r1.hpp

```diff
@@ -42,6 +42,48 @@ struct FqParams {
     static constexpr uint64_t primitive_root_1 = 0UL;
     static constexpr uint64_t primitive_root_2 = 0UL;
     static constexpr uint64_t primitive_root_3 = 0UL;
+
```

Comment:
have been derived manually or from a known source? i think we should have some scripts or something to verify them, wdyt?